static/lang/en.json

```diff
@@ -3932,7 +3934,6 @@
         "TraitDescriptionWitch": "This indicates abilities from the witch class.",
         "TraitDescriptionWizard": "This indicates abilities from the wizard class.",
         "TraitDescriptionWood": "Effects with the wood trait conjure or manipulate wood. Those that manipulate wood have no effect in an area without wood. Creatures with this trait consist primarily of wood or have a connection to magical wood.",
-        "TraitBottledBreath": "Bottled Breath",
         "TraitCenser": "Censer",
```

Comment:
TraitCenser too.

Reply:
Damn, knew I was going to forget one. We can save that for a later PR though, mostly it's the riders on this that I want in.